rnd/autogpt_builder/src/components/Flow.tsx

```diff
@@ -838,17 +838,17 @@ const FlowEditor: React.FC<{
   const editorControls: Control[] = [
     {
       label: "Undo",
-      icon: <Undo2 />,
+      icon: <Undo2 size={18} />,
```

Comment:
The size for the icons in the control panel would be set in the ControlPanel.tsx. We would just want to pass in which Icon we prefer to use here.

Reply:
I don't see a straight-forward way to do that without breaking support for some icon sets, do you?

Reply:
We need to let the container of the icon determine its size. We should ideally not be setting so many styles and sizing constraints in place and letting our UI components tell the story for us.